src/widgets/event/ui/TicketInfo.tsx

```
@@ -46,18 +46,17 @@ const TicketInfo = ({ eventId }: { eventId: number }) => {
+      console.log('API 응답:', response);
 
       if (response.isSuccess && Array.isArray(response.result)) {
         const orderIds = response.result;
 
         navigate('/payment/ticket-confirm', { state: { orderIds, ticketId, eventId } });
       } else {
-        alert("주문 정보를 불러올 수 없습니다.");
+        alert('주문 정보를 불러올 수 없습니다.');
       }
-
     } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)